apm-configuration/node-js.mdx

```diff
@@ -139,4 +139,8 @@ sudo apt-get install g++
 sudo apt-get install make
 ```
 
+### Running Apps on Docker 
+
+If you are unable to run your Node.js app on Docker, open port `9319` locally to enable proper firewall configurations.
```

Comment:
This isn't node JS specific. It's any of our APM's that have the ability to use docker.